### ISSUE TITLE
UbiquiCard

### ISSUE THREAD

**Issue by 0x4007** [pavlovcik/uad-ui#306]
# Ubiquity Card

![Image](https://user-images.githubusercontent.com/4975670/196578548-8bea6201-df66-44a9-b7ef-65328de3840b.png)

Low priority, but I'm very curious about how easy it is to launch a card (Visa claims its pretty easy.) I would be really keen to use it myself, so I'm sure over time after adding enough features I would personally want, we could probably commercialize it. In the near-to-mid-term, we could simply gate access to UbiquiStick holders.

https://partner.visa.com/site/programs/fintech-program.html

Still need to apply (waiting on some paperwork from incorporation document holding company)

# Features of Interest

## Ubiquity Credit (uCR) Cashback Rewards Program
1. Rewards program to incentivize usage
2. We should be earning like 1-2% on purchases directly afaik, 
    1. Meaning we can take this money to back the dollar (meaning uCR should become more liquid)
3. We could model it off of Starbucks' wildly successful rewards program and even simulate "expiring stars" with uCR-NFT

## Automatic loans via Liquity/MakerDAO
1. Covers small and short term liquidity crunches for when you're out and don't want to rehypothecate funds on chain to buy something    
2. You stake $1000 in some liquid Ubiquity asset (uAD, UBQ)        
    1. Curious to know if we should put it in a vault to make yield, or if staking uAD is good enough.
3. You deposit $1000 into your Ubiquity checking account (uAD, and maybe USDC/DAI/USDT)
4. Our protocol automatically opens up a Trove/CDP for you using Ubiquity owned ETH (or maybe have a global Trove/CDP, but seems riskier)
5. You spend $1000 on your UbiquiCard and your checking account balance is $0.
6. You need to spend another $500 on your UbiquiCard, and your position is backed by the Trove/CDP ($500 of DAI/LUSD is minted and covers your debt automatically)
    1. In theory, we could also instantly mint uCR and dump it to cover the user's debt; and liquidate the user's stake on our own terms instead of when the Trove/CDP demands a liquidation. But it feels like a bad idea to directly mess with stabilization mechanics (minting/burning)
7. You receive a warning and maybe a small "overdraft fee" which covers the gas costs for minting the LUSD    
    1. Need to think more about what happens if ETH goes down and these Troves/CDPs get liquidated. 
    2. We might be able to back a hedge, where we spot hold ETH in the Trove/CDP on behalf of the customer, and open a short position on DYDX to cover market moves. But this will increase borrowing costs...
        1. funding rate - in the last week I've seen spikes up to 18% APR funding rate, so with a 10% extra liquidation buffer that should cover short term loans.
    4. An obvious adjustment would be that we add another 10% buffer for liquidations (instead of 90% from Liquity, going through us we allow 80% utilization)
## Privacy.com-style Virtual Card Generation 
1. To control finances better (might also be able to just use Privacy.com)

## MetaVault Savings Account 
1. Cycle between our whitelisted, and highest APR vaults to get a savings account that nets 10-20% APR on stables

## Ser:

I think it is too late to think about our own crypto card.

Better to create infrastructure for other crypto cards, enhancing user experience in the metaverse 

- Cashback and loyalty programs (like in extended vision on Lootbox)

- Yield aggregation programs (game vaults like in PLAY)

- Metaverse E-commerce programs (liquidity flows, and others)

Mastercard is actively cooperating with crypto as well
https://www.mastercard.com/global/en/business/issuers/crypto/card-program.html

Mastercard crypto accelerator 
https://developer.mastercard.com/programs/start-path-crypto/

Visa launches a card with FTX
https://cointelegraph.com/news/ftx-and-visa-partner-to-permit-crypto-payments-in-40-countries
